ps2-decompile/callbacks.c

```diff
@@ -1,194 +1,220 @@
-#include <usbd.h>
+#include "callbacks.h"
 
-#include "globals.h"
-#include "shared.h"
+static char report[0x400];
+static const char string_desc[] = {0};
+static const int off_2C30 = 0x2C30;
+static char kb[0x10];
 
 int data_transfer_done(int result, int count, shared_t *g) {
-    int usbmouse_count; // $a2
-    int v4;             // $gp
-    int i;              // [sp+14h] [+14h]
+    int i, offset;
+    (void) count;
 
+#ifdef DEBUG
     if (result)
-        printf("usbmouse%d: %s -> 0x%x\n", g->port1, "sceUsbdInterruptTransfer", result);
+        (void) printf("usbmouse%d: %s -> 0x%x\n", g->port1, "sceUsbdInterruptTransfer", result);
+#endif
+
     if (!result && g->descriptorChecksPassed) {
-        usbmouse_count = g->count + 1;
-        g->count = usbmouse_count;
-        printf("usbmouse%d: count=%d data=(", g->port1, usbmouse_count);
+        g->count += 1;
```

Comment:
Why are the above lines deleted? I do not want printf calls removed.

Reply:
Opps, forgot to put it within an ifdef. ill fix that.